jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/PluginConfigurationProcessor.java

```diff
@@ -36,23 +36,23 @@
 import org.gradle.api.GradleException;
 
 /** Configures and provides builders for the image building tasks. */
-class BuildTaskConfigurator {
+class PluginConfigurationProcessor {
 
   /** {@code User-Agent} header suffix to send to the registry. */
   private static final String USER_AGENT_SUFFIX = "jib-gradle-plugin";
 
   /**
-   * Sets up configuration that is common among the image building tasks. This includes setting up
-   * the base image reference/authorization, container configuration, cache configuration, and layer
-   * configuration.
+   * Sets up {@link BuildConfiguration} that is common among the image building tasks. This includes
+   * setting up the base image reference/authorization, container configuration, cache
+   * configuration, and layer configuration.
    *
    * @param logger the logger used to display messages.
    * @param jibExtension the {@link JibExtension} providing the configuration data
    * @param projectProperties used for providing additional information
-   * @return a new {@link BuildTaskConfigurator} containing pre-configured builders
+   * @return a new {@link PluginConfigurationProcessor} containing pre-configured builders
    * @throws InvalidImageReferenceException if parsing the base image configuration fails
    */
-  static BuildTaskConfigurator newCommonConfiguration(
+  static PluginConfigurationProcessor newCommonConfiguration(
```

Comment:
perhaps this should be name along the lines of `processCommonConfiguration`? `newCommonConfiguration` may give the meaning of constructing an object of type `CommonConfiguration`.